ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/altair/statetransition/epoch/EpochProcessorAltair.java

```diff
@@ -83,7 +83,7 @@ public void processParticipationUpdates(final MutableBeaconState genericState) {
     state.getPreviousEpochParticipation().setAll(state.getCurrentEpochParticipation());
 
     // Reset current epoch participation flags
-    state.getCurrentEpochParticipation().clear();
+    state.clearCurrentEpochParticipation();
     state.getCurrentEpochParticipation().setAll(SszByte.ZERO, state.getValidators().size());
```

Comment:
(future possible optimization): create a specific `SszByteList` and `SszByteListSchema` implementations with `SszByteListSchema.create(byte defaultValue, int size)` where you may directly create leaf tree nodes in a faster way. 
This might make sense as this list could be quite large.